setup.py

```diff
@@ -128,6 +128,8 @@ def check_versions(show_only=False):
                   show_only=show_only)
     package_check('primme', INFO.PRIMME_MIN_VERSION, optional=True,
                   version_getter=_primme_version, show_only=show_only)
+    package_check('oct2py', INFO.OCT2PY_MIN_VERSION, optional=True,
+                  version_getter=_primme_version, show_only=show_only)
```

Comment:
The (wrong) `version_getter` should be removed, as `oct2py.__version__` works fine. :)